docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -189,6 +190,95 @@ spec:
   query: select * from processes
 ```
 
+## Using fleetctl with an API-only user
+
+Now that `fleetctl` and the Fleet server are configured, it can be helpful to create an API-only user to use when running automated workflows. An API-only user can be given a role based on the abilities it needs. The default access level is `Observer`. For more information on roles, see the [user permissions documentation](./Permissions.md#user-permissions).
+
+### Create an API-only user
+
+To create your new API-only user using `fleetctl user create` and passing a name, email and password and setting the `api-only` flag to `true`:
+
+```
+fleetctl user create --name "API User" --email api@example.com --password temp!pass --global-role observer --api-only true
+```
+
+### Reset the password
```

Comment:
This is awesome that you ran into these extra steps.

> Was adding info to the docs for API-only users and figured I could fix #4540 at the same time rather than needing to go back and edit the docs later.

I pasted your comment from [Slack](https://fleetdm.slack.com/archives/C02A8BRABB5/p1647529013379549) and responded below because I think it applies to these extra "Reset password" steps.

I propose that the doc improvements in this PR get merged before addressing #4540. This way, you can ship these doc improvements quickly so we can start pointing user's towards them.

After the docs are merged in, I think it totally makes sense to address #4540. 

I updated the description of #4540 so that it only includes the task of updating the `fleetctl user create --api-only` command so that it always creates a user with `admin_forced_password_reset` set to false. 

This way, we can make a smaller, iterative change to cut out the "Reset password" steps that you've documented. Later, we might want to add the additional flags the community member suggested.

I also looped in a member of the engineering team to make sure that there are no security concerns with this change.

Again, it's so sweet that documenting these steps resulted in encountering unnecessary steps.

Reply:
That makes a lot of sense!

Reply:
>documenting these steps resulted in encountering unnecessary steps.

Yes! That is half the value of writing things down sometimes